app/main.py

```diff
@@ -1,8 +1,38 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name, age, wife=None, husband=None):
+        self.name = name
+        self.age = age
+
+        if husband is not None:
+            self.husband = husband
+        if wife is not None:
+            self.wife = wife
```

Comment:
you do double or even triple work. Let your constructor take just name and age. Then you can rewrite your create_person_list function and avoid some checks and attribute assigns.